docs/docusaurus/docs/core/introduction/try_gx.md

```diff
@@ -55,9 +55,18 @@ This example workflow walks you through connecting to data in a Pandas DataFrame
 This example requires that [Pandas](https://pandas.pydata.org/) is installed in the same Python environment where you are running GX Core.
 :::
 
-<Tabs>
+### Procedure
```

Comment:
@Rachel-Reverie Small nit, is there any way we can restrict the small rendered nav TOC on the right to only show the H2 headings? The **Procedure** headings look kind of clunky in there, and I see on other pages, like _Connect to SQL data_ the righthand nav only shows the top level sections.

![Screenshot 2024-08-23 at 9 42 52 AM](https://github.com/user-attachments/assets/53f2b51f-c556-4235-9ad0-45eb44b7d2ad)

Reply:
We can.  I'll add an edit to restrict those to H2.